CodeEdit/Features/CEWorkspace/Models/CEWorkspaceFileManager+FileManagement.swift

```diff
@@ -129,30 +110,11 @@ extension CEWorkspaceFileManager {
     public func delete(file: CEWorkspaceFile, confirmDelete: Bool = true) {
         // This function also has to account for how the
         // - file system can change outside of the editor
-        let messageText: String
-        let informativeText: String
-
-        if !file.isFolder || file.isEmptyFolder { // if its a file or an empty folder, call it by its name
-            messageText = "Are you sure you want to delete \"\(file.name)\"?"
-            informativeText = "This item will be deleted immediately. You can't undo this action."
-        } else {
-            let childrenCount = try? fileManager.contentsOfDirectory(
-                at: file.url,
-                includingPropertiesForKeys: nil
-            ).count
-
-            if let childrenCount {
-                messageText = "Are you sure you want to delete the \((childrenCount) + 1) selected items?"
-                informativeText = "\(childrenCount) items will be deleted immediately. You can't undo this action."
-            } else {
-                messageText = "Are you sure you want to delete the selected items?"
-                informativeText = "Selected items will be deleted immediately. You can't undo this action."
-            }
-        }
+        let message = file.name
 
         let deleteConfirmation = NSAlert()
-        deleteConfirmation.messageText = messageText
-        deleteConfirmation.informativeText = informativeText
+        deleteConfirmation.messageText = "Do you want to delete \(message)?"
```

Comment:
We should put quotes around the file name. Make sure to use left quote and right quote characters.